### ISSUE TITLE
[Accessibility]: Code monitoring logs screen reader issues

### ISSUE THREAD

**Issue by limitedmage** [sourcegraph/sourcegraph#36513]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34194

### Problem description

- Buttons to expand should not have aria-label. Instead, the label should be on the expand/collapse icon.
- "Monitor details"  and "N new results" links should be outside the button and the icon should read "Open in a new tab"
- SUCESS/SKIPPED/ERROR tags should have aria-label so they are not read as uppercase. The label should read ", monitor state: <state>"
- Add hidden "Monitor name:" before every name and "Last run:" before every timestamp on the left column

![CleanShot 2022-06-02 at 15 37 05](https://user-images.githubusercontent.com/206864/171750198-7d071c43-00e8-47d1-90b7-75a912f51446.png)

### Expected behavior

Issues resolved as described above.

### Additional details

#35151 and #35158 are related issues that should probably be fixed first.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart. I have assigned the label 'gitstart' and added the assignee 'gitstart-sourcegraph'.

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.
